### ISSUE TITLE
Failure during docker container build: No module named 'six'

### ISSUE THREAD

**Issue by maxgerhardt** [atomic14/platformio-tensorflow-lite#2]
Cloning the repo and following the instructions for a ESP-IDF v4.0 build and Tensorflow master branch gives.

```
Finished patching kissfft
+ echo 'Finished patching kissfft'
Traceback (most recent call last):
  File "tensorflow/lite/micro/tools/make/transform_source.py", line 30, in <module>
    import six
ModuleNotFoundError: No module named 'six'
make: *** [tensorflow/lite/micro/tools/make/gen/esp_xtensa-esp32_default/prj/hello_world/esp-idf/main/esp/main.cc] Error 1
tensorflow/lite/micro/examples/hello_world/Makefile.inc:43: recipe for target 'tensorflow/lite/micro/tools/make/gen/esp_xtensa-esp32_default/prj/hello_world/esp-idf/main/esp/main.cc' failed
The command '/bin/sh -c IDF_PATH="/src/esp/esp-idf" . esp/esp-idf/export.sh &&     cd tensorflow &&     IDF_PATH="/src/esp/esp-idf" make -f tensorflow/lite/micro/tools/make/Makefile TARGET=esp generate_hello_world_esp_project' returned a non-zero code: 2
```

Full log is at https://pastebin.com/Urfi8znG

I'll try and patch the Dockerfile with pip install six..

**Comment by maxgerhardt** [comment]
Weird: Even if I put `RUN pip3 install six` in the Dockerfile, it says

```
 ---> Running in 8da7a8baa2b0
Requirement already satisfied: six in /usr/lib/python3/dist-packages
Removing intermediate container 8da7a8baa2b0
 ---> 9251c5ca5ad7
Step 10/15 : RUN pip install six
 ---> Running in a3bd3d76c5fb
Requirement already satisfied: six in /usr/lib/python3/dist-packages
Removing intermediate container a3bd3d76c5fb
```
..

**Comment by cgreening** [comment]
Weird - couldn't recreate, but I've pushed up a commit to update the docker file 51b89861ac0b07c2826fd19d66118c77357e3222

**Comment by cgreening** [comment]
Which version of the IDF and TFLite were you targeting?

**Comment by maxgerhardt** [comment]
As per pastebin log and `README`, the command was

>sudo docker build . -t tflite-generator --build-arg tensorflow_version=master --build-arg esp_idf_version=release/v4.0

**Comment by cgreening** [comment]
Ah sorry - trying now.

**Comment by cgreening** [comment]
Fixed with: 79da46f294cc2e0142bab693877f334d4a935bad
